src/include/kdbopmphm.h

```diff
@@ -22,17 +22,25 @@
  * opmphmRatio should never be less than 1.
```

Comment:
`opmphmRatio` is not a `#define`. It is a c variable. I think the preprocessor has no knowledge of c vars.